modules/tide_publication/tide_publication.install

```diff
@@ -278,3 +278,27 @@ function tide_publication_update_10008() {
       ->save();
   }
 }
+
+/**
+ * Update the position of field_topic for the publication type.
+ */
+function tide_publication_update_10009() {
+  /** @var \Drupal\tide_core\TideEntityUpdateHelper $tideEntityUpdateHelper */
+  $tideEntityUpdateHelper = \Drupal::service('tide_core.entity_update_helper');
+  $config = \Drupal::configFactory()->getEditable('core.entity_form_display.node.publication.default');
+  $data = $config->getRawData();
+  $field_topic = [
+    'settings' => [
+      'match_operator' => 'CONTAINS',
+      'size' => 60,
+      'placeholder' => '',
+      'match_limit' => 10,
+    ],
+    'third_party_settings' => [],
+    'type' => 'entity_reference_autocomplete',
+    'region' => 'content',
+  ];
+  $data['content'] = $tideEntityUpdateHelper->repositionComponentsRelativeToKey($data['content'], ['field_topic' => $field_topic], 'field_content_category', 'before');
```

Comment:
Here’s an example of how to use `repositionComponentsRelativeToKey`. Since it doesn’t involve grouped fields,  we can just use `repositionComponentsRelativeToKey` to position a field in an `entity_form_display` entity.